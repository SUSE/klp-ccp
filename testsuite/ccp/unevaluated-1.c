int espr_o;

void pu_f(void)
{
	sizeof(espr_o), __alignof__(espr_o);
}
