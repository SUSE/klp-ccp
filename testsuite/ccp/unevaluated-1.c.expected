/* klp-ccp: from testsuite/ccp/unevaluated-1.c */
extern int espr_o;

void klpp_pu_f(void)
{
	sizeof(espr_o), __alignof__(espr_o);
}
